utils/get_supported_scylla_base_versions.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+import os
+
+from pkg_resources import parse_version
+from sdcm.utils.version_utils import is_enterprise
+from sdcm.utils.common import get_s3_scylla_repos_mapping
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+
+
+# We support to migrate from specific OOS version to enterprise
```

Comment:
Typo, must be OSS -> Open Source Scylla.

Reply:
fixed.